gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/ExecutionTreeBuilder.java

```diff
@@ -78,12 +78,13 @@ public QueryNode build() {
     // the data store
     if (singleSourceForAllAttributes.isPresent()) {
       String source = singleSourceForAllAttributes.get();
-      QueryNode selectionAndFilterNode = buildExecutionTreeForSameSourceFilterAndSelection(source);
+      QueryNode rootNode = buildExecutionTreeForSameSourceFilterAndSelection(source);
       if (LOG.isDebugEnabled()) {
-        LOG.debug("Execution Tree:{}", selectionAndFilterNode.acceptVisitor(new PrintVisitor()));
+        LOG.debug("Execution Tree:{}", rootNode.acceptVisitor(new PrintVisitor()));
```

Comment:
Not related to this change but can we add a similar log for the above `if (entitiesRequest.getIncludeNonLiveEntities())` condition too